.github/workflows/cluster-init.yaml

```diff
@@ -14,8 +14,6 @@ on:
 
   # Automatically run CI on branches, that have active PR opened
   pull_request:
-    branches:
-      - main
```

Comment:
Would it be better to instead add the `upgrade-knative` into the list and TODO about why is it there and when should it be removed?

Reply:
I was thinking of this as well but I'm not sure which is better. On one hand you could have other target branches to merge into if we are supporting a different release branch. But since we aren't doing that we can go with this approach of targeting the upgrade-knative branch.

Reply:
I changed this to main + upgrade-knative.